ml/Host.cc

```diff
@@ -282,6 +282,11 @@ void RrdHost::getConfigAsJson(nlohmann::json &Json) const {
     Json["idle-window-size"] = Cfg.ADIdleWindowSize;
     Json["window-rate-threshold"] = Cfg.ADWindowRateThreshold;
     Json["dimension-rate-threshold"] = Cfg.ADDimensionRateThreshold;
+
+    Json["save-anomaly-percentage-every"] = Cfg.SaveAnomalyPercentageEvery;
+
+    Json["max_anomaly_rate_info_table_size"] = Cfg.MaxAnomalyRateInfoTableSize;
+    Json["max_anomaly_rate_info_age"] = Cfg.MaxAnomalyRateInfoAge;
```

Comment:
Use - instead of _ to be consistent with the rest of the fields.

Reply:
Done.